xilem/src/view/button.rs

```diff
@@ -50,7 +52,14 @@ where
     type ViewState = ();
 
     fn build(&self, ctx: &mut ViewCtx) -> (Self::Element, Self::ViewState) {
-        ctx.with_leaf_action_widget(|ctx| ctx.new_pod(widget::Button::new(self.label.clone())))
+        ctx.with_leaf_action_widget(|ctx| ctx.new_pod(
+            widget::Button::from_label(widget::Label::new(self.label.label.clone())
```

Comment:
I think this is the best we can do with the current design of `xilem::Pod`, but it's not a good final state.

Reply:
Okey. So, I do not need to change this code yet..